src/__stories__/JsonSchemaViewer.tsx

```diff
@@ -159,19 +160,26 @@ storiesOf('JsonSchemaViewer/combiners', module)
       onGoToRef={action('onGoToRef')}
     />
   ))
-  .add('anyOf-array-schema', () => (
+  .add('oneOf-array-schema', () => (
```

Comment:
this change is unrelated to this issue, but I also updated the story names to match the files/schema